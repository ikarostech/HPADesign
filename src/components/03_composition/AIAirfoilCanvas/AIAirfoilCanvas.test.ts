import { mount } from '@vue/test-utils';
import AIAirfoilCanvas from './AIAirfoilCanvas.vue';