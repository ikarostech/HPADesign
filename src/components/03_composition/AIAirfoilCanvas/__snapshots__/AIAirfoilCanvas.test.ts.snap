// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AIAirfoilCanvas snapshot test 1`] = `<div />`;
